fdbsyncd/fdbsyncd.cpp

```diff
@@ -16,10 +16,10 @@ int main(int argc, char **argv)
 {
     Logger::linkToDbNative("fdbsyncd");
 
-    DBConnector appDb(APPL_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
+    DBConnector appDb("APPL_DB", 0);
```

Comment:
Just want to double check you are using unix socket. We are moving away from TCP port due to performance and security concerns.

Reply:
Yes, I am using a Unix socket.
src/sonic-swss-common/common/dbconnector.cpp lines 674 to 691.

DBConnector::DBConnector(const string& dbName, unsigned int timeout, bool isTcpConn, const SonicDBKey &key)
    : m_dbId(SonicDBConfig::getDbId(dbName, key))
    , m_dbName(dbName)
    , m_key(key)
{
    struct timeval tv = {0, (suseconds_t)timeout * 1000};
    struct timeval *ptv = timeout ? &tv : NULL;
    if (isTcpConn)
    {
        initContext(SonicDBConfig::getDbHostname(dbName, m_key).c_str(), SonicDBConfig::getDbPort(dbName, m_key), ptv);
    }
    else
    {
        initContext(SonicDBConfig::getDbSock(dbName, m_key).c_str(), ptv);
    }

    select(this);
}

Reply:
The code here actually invoke this ctor, the default value of isTcpConn is false, so it's using unix socket:

    DBConnector(const std::string &dbName, unsigned int timeout, bool isTcpConn = false);